python_modules/dagster/dagster/_core/execution/job_execution_result.py

```diff
@@ -138,13 +141,18 @@ def _get_output_for_handle(self, handle: NodeHandle, output_name: str) -> Any:
             f"Did not find result {output_name} in {node.describe_node()}"
         )
 
-    def _get_value(self, context, step_output_data, dagster_type):
+    def _get_value(
+        self,
+        context: StepExecutionContext,
+        step_output_data: StepOutputData,
+        dagster_type: DagsterType,
+    ) -> object:
         step_output_handle = step_output_data.step_output_handle
         manager = context.get_io_manager(step_output_handle)
         manager_key = context.execution_plan.get_manager_key(step_output_handle, self.job_def)
         res = manager.load_input(
             context.for_input_manager(
-                name=None,
+                name="dummy_input_name",
```

Comment:
so the context here is that we are loading an output as an input just to get the value so the input name should almost certainly not matter, does that sound right?

Reply:
maybe a more specific placeholder name that provides some context in case someone has io manager code that interacts with input name? seems pretty in the weeds tho

Reply:
exactly, which seems to be empirically born out by the fact that `None` worked here even when a string was expected